README.md

```diff
@@ -62,3 +62,6 @@ Feel free to email us at [support@status.im](mailto:support@status.im) or better
 ## License
 
 Licensed under the [Mozilla Public License v2.0](https://github.com/status-im/status-react/blob/develop/LICENSE.md)
+
+## Testing Supported by<br/>
```

Comment:
don't need br in markdown

Reply:
If you add a space between header and body it should look fine I think

Reply:
br has been removed